README.md

```diff
@@ -75,77 +75,118 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+
+#### Highlights from v0.45
 
 **General**
 
-- New UI for sizes list view in Image Resizer settings. Thanks @niels9001!
-- Fixed FileInUse errors during install/update scenarios.
-- Fixed toggle switches on PowerToys run settings to display correctly.
-- Fixed header text not updating when theme color is changed. Thanks @niels9001!
-- Changed tooltip text for systray icon to be on a single line for Windows 11 compatibility.
-- Expanded the Report Bug tool to collect more robust diagnostic information.
-- Fixed screen reader functionality to stop announcing hidden text in settings.
+- Settings and OOBE windows updated with Fluent UX! We hope you enjoy the new modern feel of the application menus as we align our product with upcoming Windows 11 interfaces. Special thanks to @niels9001 for driving the development of this UI, along with many thanks to the various community members who offered constant feedback and adjustments to make this a truly spectacular update!
+- Added button to settings which displays version history. Thanks @niels9001!
+- Signed PowerToysSetupCustomActions.dll.
+- Improved Update available & Update ready message severity visual. Thanks @niels9001!
+- Improved auto-update experience. Thanks @niels9001!
+- Aligned OOBE theme color with Settings theme color. Thanks @niels9001!
+- Adjusted labeling of "Restart as Administrator" button to "Restart PowerToys as Administrator" to avoid ambiguity in meaning. Thanks @niels9001!
+- Added colored icons to settings sidebar. Thanks @niels9001!
+- Fixed accessibility issue in OOBE where Microsoft Docs and PowerToys release notes links could not be navigated to via keyboard. Thanks @niels9001!
+- Fixed settings header alignment. Thanks @niels9001!
+- Fixed updates text next being visible when in light mode. Thanks @niels9001!
+- Updated "Learn More" text to be more descriptive. Thanks @niels9001!
+- Updated "Read more" text on updates to be more descriptive. Thanks @niels9001!
+- Added link to documentation in system tray. Thanks @BenConstable9!
+- Fixed error caused by file in use issues when installing PowerToys.
+- Fixed issue where opening settings from start menu didn't work when PowerToys was run as admin. Thanks @davidegiacometti!
  
 ### Awake
 
-- Fixed bug when right-clicking menu of Awake app icon. Thanks @dend!
-- Fixed high CPU usage for timed keep awake. Thanks @dend!
-- Fixed Awake icon spamming notification tray. Thanks @dend!
-- Added telemetry to collect Awake settings and logs.
+- Added Awake as option in translation bug template. Thanks @Aaron-Junker!
+- Adjusted description of inactive setting to improve distinguishing between the utility being disabled vs inactive. Thanks @niels9001!
 
 ### Color Picker
 
-- Fixed escape behavior so that only the fly-out is closed if active.
-- Fixed several accessibility issues related to screen reader functionality and general usage of Color Picker.
+- Fixed bug where changing RGB values doesn't update color. Thanks @niels9001!
```

Comment:
Actually @martinchrzan is the one who made a pr for this

Reply:
Thanks for the callout, I mixed up Niels who the issue was assigned to with Martin who actually created the PR.